### ISSUE TITLE
Get the current section name and progression

### ISSUE THREAD

**Issue by llemeurfr** [edrlab/thorium-reader#570]
A blind user should be able to determine both current section name and page number/percentage read without losing the reading position.

TODO: study the keyboard shortcut which could trigger that info.

**Comment by danielweck** [comment]
Related issue (long conversation thread): https://github.com/edrlab/thorium-reader/issues/1004

**Comment by danielweck** [comment]
Implemented.
